superset/Dockerfile

```diff
@@ -105,22 +106,37 @@ ENV FLASK_APP="superset.app:create_app()" \
 ENV PATH="${HOME}/app/bin:${PATH}" \
     PYTHONPATH="${HOME}/app/pythonpath"
 
-RUN microdnf update \
-    && microdnf install \
-        cyrus-sasl \
-        openldap \
-        openldap-clients \
-        openssl-libs \
-        openssl-pkcs11 \
-        python${PYTHON} \
-    && microdnf clean all && \
-    rm -rf /var/cache/yum
+RUN <<EOF
+microdnf update
+microdnf install \
+  cyrus-sasl \
+  openldap \
+  openldap-clients \
+  openssl-libs \
+  openssl-pkcs11 \
+  python${PYTHON}
+
+microdnf clean all
+rm -rf /var/cache/yum
+
+# All files and folders owned by root group to support running as arbitrary users.
+# This is best practice as all container users will belong to the root group (0).
+chown -R ${STACKABLE_USER_UID}:0 /stackable
```

Comment:
```suggestion
chown -R "${STACKABLE_USER_UID}:0" /stackable
```

Reply:
Same applies to any others.
Feel free to ignore if the previous changes also didn't do this. We can clean it up in a later PR.

Reply:
The previous changes also didn't do this but that doesn't mean we can't do it here. At least for your suggestions as that's already done.

Reply:
That said: These are replaced at the "Docker" level before it even gets to bash so I don't think it's super important in this case.

Reply:
Oh, good point!